bin/input_masterlist_ml.yaml

```diff
@@ -0,0 +1,130 @@
+defaultTimeOut: 500 #customize the default timeout threshold
+
+## To inject faults into all the layers of the model:
+## LayerNo: '0' indicates all layers of the type mentioned in LayerName; For layerName, you can specify 'all' to inject fault in all the layers
+compileOption:
+    instSelMethod:
+      - customInstselector:
+          include:
+            - CustomTensorOperator
+          options:
+            -layerNo=0 
+            -layerName=all
+
+## To inject faults into all layers of a specific type in the model:
+## LayerNo: '0' indicates all layers of the type mentioned in LayerName; Mention the type of layer(convolution, relu etc) in layerName. For the below example, fault injection is done for all convolution and relu layers.
+compileOption:
+    instSelMethod:
+      - customInstselector:
+          include:
+            - CustomTensorOperator
+          options:
+            -layerNo=0;0;0;0;0;0;0 # 
+            -layerName=conv;relu
+            
+## To inject faults into a specific layer number in the model:
+## LayerNo: '1' indicates first layer of the 'layerName' type; In the below example, it is the first convolution layer in the model
```

Comment:
Is the `LayerNo` a zero-based or one-based index?

Reply:
```LayerNo``` is one-based index